AutoResizeFont/ViewController.swift

```diff
@@ -9,7 +9,9 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet var label1 : DynamicLabel!
```

Comment:
We still have this outlet.
I'd prefer if this file is unchanged, don't see a need to change anything here.

Reply:
fixed in commit #b25317a